### DISCUSSION TITLE
Default value for “message”

### DISCUSSION THREAD

**Discussion by fredericalpers** [onOffice-Web-Org/oo-wp-plugin#481] [_Discarded pitches]
### Current state

If the default value is filled in, the success message appears after sending the form, but no e-mail is received and no address is created. It is particularly bad if the request for more information on a property is not received. The bug seems to occurred first with plugin version 4.9. Plugin Versions 4.6 - 4.8. seem to work fine.

### Desired state
Investigate if the bug only occurs with the “message” field or if there is a problem with more fields. Currently I saw that the “message” field has this bug in the normal contact form as well as the contact form on property detail pages.

The e-mail must be received and an address created (if the box is checked in the WP Backend) even if there's a default value for the “message” field and/or other fields.

![default_value_message](https://user-images.githubusercontent.com/88546396/229791678-5652d5f1-4c2f-4adb-a171-3d6b3500455a.jpg)

**Comment by fredericalpers** [comment]
@dai-eastgate Please investigate and let me know how long a fix will approx. take.

**Reply by dai-eastgate** [reply]
@fredericalpers I need a day to fix and test it.

**Reply by dai-eastgate** [reply]
Added to this issue https://github.com/onOffice-Web-Org/oo-wp-plugin/issues/488
